components/menu/index.css

```diff
@@ -11,849 +11,945 @@ governing permissions and limitations under the License.
+
+	--spectrum-menu-item-label-content-color-default: var(--spectrum-neutral-content-color-default);
+	--spectrum-menu-item-label-content-color-hover: var(--spectrum-neutral-content-color-hover);
+	--spectrum-menu-item-label-content-color-down: var(--spectrum-neutral-content-color-down);
+	--spectrum-menu-item-label-content-color-focus: var(--spectrum-neutral-content-color-key-focus);
+
+	--spectrum-menu-item-label-icon-color-default: var(--spectrum-neutral-content-color-default);
+	--spectrum-menu-item-label-icon-color-hover: var(--spectrum-neutral-content-color-hover);
+	--spectrum-menu-item-label-icon-color-down: var(--spectrum-neutral-content-color-down);
+	--spectrum-menu-item-label-icon-color-focus: var(--spectrum-neutral-content-color-key-focus);
+
+	--spectrum-menu-item-label-content-color-disabled: var(--spectrum-disabled-content-color);
+	--spectrum-menu-item-label-icon-color-disabled: var(--spectrum-disabled-content-color);
+	--spectrum-menu-item-thumbnail-opacity-disabled: 0.3;
+
+	--spectrum-menu-item-label-font-weight: var(--spectrum-medium-font-weight);
+	--spectrum-menu-item-description-font-weight: var(--spectrum-regular-font-weight);
+
+	--spectrum-menu-item-description-line-height: var(--spectrum-line-height-100);
+	--spectrum-menu-item-description-line-height-cjk: var(--spectrum-cjk-line-height-100);
+
+	--spectrum-menu-item-description-color-default: var(--spectrum-neutral-subdued-content-color-default);
+	--spectrum-menu-item-description-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+	--spectrum-menu-item-description-color-down: var(--spectrum-neutral-subdued-content-color-down);
+	--spectrum-menu-item-description-color-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+	--spectrum-menu-item-description-color-disabled: var(--spectrum-disabled-content-color);
+
+	--spectrum-menu-section-header-line-height: var(--spectrum-line-height-100);
+	--spectrum-menu-section-header-line-height-cjk: var(--spectrum-cjk-line-height-100);
+	--spectrum-menu-section-header-font-weight: var(--spectrum-bold-font-weight);
+	--spectrum-menu-section-header-color: var(--spectrum-neutral-content-color-default);
+	--spectrum-menu-section-description-line-height: var(--spectrum-line-height-100);
+	--spectrum-menu-section-description-line-height-cjk: var(--spectrum-cjk-line-height-100);
+	--spectrum-menu-section-description-font-weight: var(--spectrum-regular-font-weight);
+	--spectrum-menu-section-description-color: var(--spectrum-neutral-subdued-content-color-default);
+	--spectrum-menu-collapsible-icon-color: var(--spectrum-gray-900);
+
+	--spectrum-menu-checkmark-icon-color-default: var(--spectrum-accent-content-color-default);
+	--spectrum-menu-checkmark-icon-color-hover: var(--spectrum-accent-content-color-hover);
+	--spectrum-menu-checkmark-icon-color-down: var(--spectrum-accent-content-color-down);
+	--spectrum-menu-checkmark-icon-color-focus: var(--spectrum-accent-content-color-key-focus);
```

Comment:
Should our property be `--spectrum-menu-checkmark-icon-color-key-focus`? They weren't like that before, I'm just thinking of making it all consistent.

Reply:
Oh and does this "checkmark" chunk need a "disabled" property? To use the `disabled-content-color` token? I don't have a good grasp on how `.is-disabled` is affecting menu items like a checkmark though...

Reply:
🤔 We didn't use `key-focus` for any of the property definitions before, so I'm hesitant to change it in the migration, but I spy one `key-focus` property definition that I added in that branch, I'm wondering if maybe I should remove that one.

I didn't add a disabled icon color, it looks like the disabled color is coming through from the `.spectrum-Menu-itemIcon` class being set on it and all the icons are set to `--spectrum-menu-item-label-icon-color-disabled`.